packages/client-search/src/types/SearchOptions.ts

```diff
@@ -373,4 +373,11 @@ export type SearchOptions = {
    * Sets the languages to be used by language-specific settings and functionalities such as ignorePlurals, removeStopWords, and CJK word-detection.
    */
   readonly queryLanguages?: readonly string[];
+
+  /**
+   * Enriches the API’s response with meta-information as to how the query was processed. This attirbute powers the 'Query Alternatives' explanation.
```

Comment:
```suggestion
   * Enriches the API’s response with meta-information as to how the query was processed.
```

it's for more than query alternatives